pootle/apps/contact/forms.py

```diff
@@ -56,6 +57,17 @@ def from_email(self):
     def recipient_list(self):
         return [settings.POOTLE_CONTACT_EMAIL]
 
+    def save(self, fail_silently=False):
+        """
+        Build and send the email message.
+
+        """
+        kwargs = self.get_message_dict()
+        kwargs["headers"] = {"reply-to": kwargs["from_email"]}
+        kwargs["from_email"] = settings.DEFAULT_FROM_EMAIL
```

Comment:
Just wondering if this would be easier to read if it is an update call:

```python
    kwargs.update({
        'headers': {
            'Reply-To': kwargs['from_email'],
        },
        'from_email': settings.DEFAULT_FROM_EMAIL,
    })
```